Assets/Prefabs/Level End Trigger.prefab

```diff
@@ -0,0 +1,126 @@
+%YAML 1.1
```

Comment:
I think the Sprite Renderer component should be removed from this prefab. The bounds are already visible in the Editor because of the Box Collider 2D component.